packages/dhis2/src/Adaptor.js

```diff
@@ -485,10 +542,10 @@ export function get(resourceType, query, options = {}, callback = false) {
  * @returns {Operation}
  * @example <caption>Example `expression.js` of upsert</caption>
  * upsert('trackedEntityInstances', {
- *  ou: 'TSyzvBiovKh',
+ *  orgUnits: 'TSyzvBiovKh',
  *  filter: ['w75KJ2mc4zz:Eq:Qassim'],
  * }, {
- *  orgUnit: 'TSyzvBiovKh',
+ *  orgUnits: 'TSyzvBiovKh',
```

Comment:
If I look at the [docs](https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-241/tracker.html#tracked-entities), they say `orgUnit` (singular). Can we double check? and make sure this is accurate?